src/explore-education-statistics-admin/src/pages/users/components/InviteUserPublicationRoleForm.tsx

```diff
@@ -122,13 +122,13 @@ const InviteUserPublicationRoleForm = ({
                     <tr
                       key={`${userPublicationRole.publicationId}_${userPublicationRole.publicationRole}`}
                     >
-                      <td>
+                      <th>
```

Comment:
This was for the UI tests - we have several utility keywords that look for a row heading - so rather than write new keywords I used this.

Preferably, I'll keep the row heading, but unbold the text - I'll give that a try.

Reply:
Use a testid, not a `th`

Reply:
Discussed with Nick on Slack - changed the `th` to a `td` and created a new UI tests keyword to cover it.